src/shared/parameter/config_definitions/tbots_gtest_main_command_line_args.yaml

```diff
@@ -16,3 +16,15 @@
     description: >-
         The directory to output logs to. Absolute paths are recommended as the working directory
         is inside the bazel-out directory.
+
+- int:
```

Comment:
hm, what if we made this a double and have the range between 0.1 and 10, where 0.1 is 0.1X speed, i.e. 10X slower, and 10 is 10X faster?

Reply:
Yes, that will be useful, I'll include that